### PR TITLE
link/kprobe: Allow to parse symbol offset

### PR DESCRIPTION
**Pull request by nickcooper-zhangtonghao**
Signed-off-by: Tonghao Zhang <zhangtonghao@didiglobal.com>

### CODE REVIEW

**Comment by ti-mo** [comment]
@nickcooper-zhangtonghao please avoid recreating pull requests in the future, as this makes review context get lost. GitHub allows (force)pushing to an existing PR branch.

File: link/kprobe.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ type KprobeOptions struct {
 	//
 	// Needs kernel 5.15+.
 	Cookie uint64
+	Offset uint64
```
Comment (mmat11) [inline link/kprobe.go:52]: Could you add a comment which describes what this is?

File: link/kprobe.go [hunk 2]
```diff
@@ -489,3 +495,14 @@ func kretprobeBit() (uint64, error) {
 	})
 	return kprobeRetprobeBit.value, kprobeRetprobeBit.err
 }
+
+// kprobeToken creates the SYM[+offs] token for the tracefs api.
+func kprobeToken(args probeArgs) string {
+	po := fmt.Sprintf("%s", args.symbol)
```
Comment (mmat11) [inline link/kprobe.go:501]: golangci-lint fails, `args.symbol` is already a string

**Comment by nickcooper-zhangtonghao** [comment]
>

File: link/kprobe.go (hunk 1 above)
Comment (nickcooper-zhangtonghao) [reply link/kprobe.go:52]: ok

File: link/kprobe.go (hunk 2 above)
Comment (nickcooper-zhangtonghao) [reply link/kprobe.go:501]: ok

**Pull request closed without merging**